x-pack/legacy/plugins/apm/public/components/app/Settings/AgentConfigurations/AgentConfigurationCreateEdit/SettingsPage/SettingsPage.tsx

```diff
@@ -0,0 +1,233 @@
+                    })}
+                  </EuiButton>
+                )}
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </EuiPanel>
+
+          <EuiSpacer size="m" />
+
+          {/* Settings panel */}
+          <EuiPanel paddingSize="m">
+            <EuiTitle size="s">
+              <h3>
+                {i18n.translate('xpack.apm.agentConfig.settings.title', {
+                  defaultMessage: 'Core configuration options'
+                })}
+              </h3>
+            </EuiTitle>
+
+            <EuiSpacer size="m" />
+
+            {configSettingDefinitions
+
+              // filter out agent specific items that are not applicable
+              // to the selected service
+              .filter(filterByAgent(newConfig.agent_name as AgentName))
+              .map(setting => (
+                <SettingFormRow
+                  isUnsaved={unsavedChanges.hasOwnProperty(setting.key)}
+                  key={setting.key}
+                  setting={setting}
+                  value={newConfig.settings[setting.key]}
+                  onChange={(key, value) => {
+                    setNewConfig(prev => ({
+                      ...prev,
+                      settings: {
+                        ...prev.settings,
+                        [key]: value
+                      }
+                    }));
+                  }}
+                />
+              ))}
+          </EuiPanel>
+        </form>
+      </EuiForm>
+      <EuiSpacer size="xxl" />
+
+      {/* Bottom bar with save button */}
+      {unsavedChangesCount > 0 && (
+        <EuiBottomBar paddingSize="s">
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+            <EuiFlexItem
+              grow={false}
+              style={{
+                flexDirection: 'row',
+                alignItems: 'center'
+              }}
+            >
+              <EuiHealth color="warning" />
+              <EuiText>
+                {i18n.translate('xpack.apm.unsavedChanges', {
+                  defaultMessage:
+                    '{unsavedChangesCount, plural, =0{0 unsaved changes} one {1 unsaved change} other {# unsaved changes}} ',
+                  values: { unsavedChangesCount }
+                })}
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              <EuiFlexGroup justifyContent="flexEnd">
+                <EuiFlexItem grow={false}>
+                  <CancelButton color="ghost" />
+                </EuiFlexItem>
+                <EuiFlexItem grow={false}>
+                  <EuiButton
+                    onClick={handleSubmitEvent}
+                    fill
+                    isLoading={isSaving}
+                    isDisabled={!isFormValid}
+                  >
+                    {i18n.translate(
+                      'xpack.apm.agentConfig.settingsPage.saveButton',
+                      { defaultMessage: 'Save' }
```

Comment:
```suggestion
                      { defaultMessage: 'Save configuration' }
```
I reckon we might update this when/if we get the review page, since it'd make sense to indicate where you go next. In the case it'd be `Review configuration`

Reply:
Makes sense 👍